### PR TITLE
Split workout into 2 smaller ones

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/mongodb/intro/aggregation-iv/aggregation-with-accumulators-iv.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/mongodb/intro/aggregation-iv/aggregation-with-accumulators-iv.md?ref=5ffc995d8387be866da6585acade915362d9080e)

**Comment by kapnobatai137** [comment]
we should rename the files `aggregation-with-accumulators-iv` and `aggregation-with-accumulators-v` to be more explanatory of what is inside @Stefan-Stojanovic

File: mongodb/intro/aggregation-iv/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name: Aggregation IV
+
+description: More accumulators and operators to you with aggregation.
```
Comment (kapnobatai137) [inline mongodb/intro/aggregation-iv/README.md:3]: this doesn't really make sense :sweat:

Comment (Stefan-Stojanovic) [reply mongodb/intro/aggregation-iv/README.md:3]: oh, the `you` is supposed to be `use`

File: mongodb/intro/aggregation-iv/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+name: Aggregation IV
+
+description: More accumulators and operators to use with aggregation.
+
+insights:
+  - aggregation-with-accumulators-push
+  - aggregation-with-accumulators-sum
```
Comment (kapnobatai137) [inline mongodb/intro/aggregation-iv/README.md:7]: how do these sound?
```suggestion
  - aggregation-with-push
  - aggregation-with-sum
```

File: mongodb/intro/aggregation-iv/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+name: Aggregation IV
+
+description: More accumulators and operators to use with aggregation.
```
Comment (kapnobatai137) [inline mongodb/intro/aggregation-iv/README.md:3]: we should try to mention what is taught in this workout. "More accumulators and operators..." sounds pretty vague.

Maybe something like: "Pushing, summing and other accumulators."

File: mongodb/intro/README.md [hunk 4]
```diff
@@ -19,6 +19,7 @@ sections:
     - arithmetic-aggregation-i
     - arithmetic-aggregation-ii
     - aggregation-iii
+    - aggregation-iv
```
Comment (kapnobatai137) [inline mongodb/intro/README.md:22]: can we also rename these?

- aggregation-with-max-and-mind

**Comment by kapnobatai137** [comment]
Also, when making these changes, can you improve the headlines for each insight? Currently, they follow this format:
> Aggregation With Accumulators (`$max` & `$min`)

I think it would look better if we only had:
> `$max` & `$min`

this is short and to the point. try to do it for all the insights involved in this PR

**Comment by enki-hq** [comment]
Rule workout:insights failed for files: 
  - mongodb/intro/aggregation-iv/README.md

**Comment by enki-hq** [comment]
Rule workout:insights failed for files: 
  - mongodb/intro/aggregation-iv/README.md

File: mongodb/intro/README.md (hunk 4 above)
Comment (Stefan-Stojanovic) [reply mongodb/intro/README.md:22]: These are workout names, I can't rename a workout with 3-4 different operators with the name of 2. Did you mean the insights named: aggregation-with-accumulators-iii and iv?

Comment (Stefan-Stojanovic) [reply mongodb/intro/README.md:22]: I've changed the names of the insights. As for workouts, they are all regarding multiple different operators, I can't name them according to one operator in the workout unless only that one is discussed there.

**Pull request merged**